vignettes/layer.Rmd

```diff
@@ -16,4 +16,303 @@ knitr::opts_chunk$set(
 
 ```{r setup}
 library(tlang)
+library(dplyr)
+library(gt)
+library(tidyr)
+```
+
+A core design element of `tlang` is to offer the ability to layer and build template table formats that can be shared, improved, and eventually customized for a specific table. The idea being that organizations can develop a standard `tfrmt` object that can be reused across many different tables, and make it so programmers can focus on setting up the data as opposed to formatting of the table. 
+
+There are two ways that `tfrmt` objects may be layered. The preferred method is through the first argument of `tfrmt` - `tfrmt_obj`. The second, but perhaps more explicit method is using the `layer_tfrmt` function. We will go through their applications below.  
+
+## Creating a template
```

Comment:
I think we might have been a touch hasty in making this vingette. I think we are better off making a canned tfrmt. There is a lot of say about creating templates so I think there should be a separate vingette for that. Just so the purpose of this one is a bit more focused